packages/AutoTDD-Core.package/ATDDTestingEntity.class/class/statuses.st

```diff
@@ -0,0 +1,14 @@
+constants
+statuses
+
+	"The order of these statuses is not arbitrary!"
+	"This is still somewhat inelegant because the 
+	reason for this has nothing to do with this class"
+
+	^ OrderedCollection 
+		newFrom: {
+			#error.
+			#failure.
+			#pending.
+			#successful
```

Comment:
Shouldn't those be strings?

Reply:
We could make them strings, let's talk about this